### ISSUE TITLE
Mark test case as async to exclude sync methods from being tested

### ISSUE THREAD

**Issue by gijswijs** [jstransformers/test-jstransformer#38]
Maybe I'm missing something here, but I'm extending the nunjucks transformer to be able to handle async rendering as well. For this I added a renderAsync method that handles this.
I also added a test case that mimics an async template loader.
This test case (obviously) fails with the sync methods, as it should. But there's no way to exclude the methods from being tested on the async test case.
The other way around (testing the async method on sync test cases) isn't a problem, since the renderAsync handles them perfectly.

**Comment by RobLoach** [comment]
Yes, I've been troubled by this myself. We're open to switching test cases!

**Comment by gijswijs** [comment]
I'll see what I can do. I'm not making any promises yet, as I'm busy these days.

**Comment by gijswijs** [comment]
See @RobLoach it only took me a year! 😄
